### ISSUE TITLE
xml_consistency, do not report error if there are parser errors in the abap

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#997]
example:

```
./src/deps/zcl_abaplint_deps_find.clas.abap[289, 5]        - Statement does not exist in ABAPv702(or a parser error), "DATA" (parser_error)
./src/deps/zcl_abaplint_deps_find.clas.xml[1, 1]           - Class matching XML name not found in ABAP file (xml_consistency)
```
